### ISSUE TITLE
Accessibility Forms

### ISSUE THREAD

**Issue by wingdu** [fluentform/fluentform#248]
Hey,

the text box dont give out an a label correctly or an aria-label. 
The same is there with the Signpad, 

Example Signpad:
<button type='button' class='fluentform-signature-button fluentform-signature-undo'>

Correct Version is:
<button type='button' aria-label="Undo" class='fluentform-signature-button fluentform-signature-undo'>

**Comment by nkb-bd** [comment]
Thanks for contacting, we are analysing the forms accessibility. It seems some portion of your message has been cut off or missing can you please check confirm ?

**Comment by wingdu** [comment]
they are html tags... but you can check it self.. its the signature pad ..
